devbox.go

```diff
@@ -423,11 +429,11 @@ func (d *Devbox) profileBinDir() (string, error) {
 }
 
 // installMode is an enum for helping with ensurePackagesAreInstalled implementation
-type installMode string
+type installMode int
 
 const (
-	install   installMode = "install"
-	uninstall installMode = "uninstall"
+	install   installMode = iota
+	uninstall installMode = iota
```

Comment:
Not a fan of this change TBH:

I prefer to type enums as strings because they are easier to debug (example: if we need to print or log the value).
The int enums are more space efficient and make more sense where that is a concern

Reply:
I'll revert because this is unrelated to PR